apps/ui/src/views/Terms.vue

```diff
@@ -255,7 +255,7 @@ You agree to promptly notify the Foundation of any Claims and cooperate with the
 
 ## 15. Privacy notice and cookie declaration
```

Comment:
Maybe also remove the cookie reference in the title, or add something in the body that says we don't track cookies